orca-core/src/main/groovy/com/netflix/spinnaker/orca/pipeline/ExecutionRunnerSupport.java

```diff
@@ -36,13 +34,27 @@ public ExecutionRunnerSupport(Collection<StageDefinitionBuilder> stageDefinition
     this.stageDefinitionBuilders = stageDefinitionBuilders;
   }
 
-  @Override
-  public <T extends Execution> void start(T execution) throws Exception {
+  /**
+   * The default implementation plans all stages.
+   * Implementations may override this method completely or use it to plan the entire execution before running it.
+   *
+   * @param execution the execution instance with skeleton stages.
+   * @param <T>       the execution type.
+   * @throws Exception
+   */
+  @Override public <T extends Execution<T>> void start(T execution) throws Exception {
```

Comment:
Was thinking (the other day) if `start` is the appropriate name for this method vs. `plan`?

It's overridden in subclasses (right?) with a super.start(execution) and then the behavior for actually running it in spring batch.